src/app/public/modules/action-hub/action-hub-buttons.component.ts

```diff
@@ -0,0 +1,7 @@
+import { Component } from '@angular/core';
+
+@Component({
```

Comment:
We need JSDocs comments here for the documentation. 

Example: https://github.com/blackbaud/skyux-lists/blob/master/src/app/public/modules/repeater/repeater-item-title.component.ts